scss/civicrm/common/_buttons.scss

```diff
@@ -26,6 +28,9 @@ $button-border-radius: 2px;
 }
 
 .crm-container {
+
+  // "Toolbar" buttons
+  $toolbar-top-space: 6px;
```

Comment:
what's this variable? why do we need it here?

Reply:
I haven't noticed this one as it was from stylelint. I will look on that